packages/mobile/src/screens/chat-screen/ChatUserListScreen.tsx

```diff
@@ -0,0 +1,274 @@
+import { useNavigation } from 'app/hooks/useNavigation'
+import type { AppTabScreenParamList } from 'app/screens/app-screen'
+import { makeStyles } from 'app/styles'
+import { useThemeColors } from 'app/utils/theme'
+
+const { searchUsers } = searchUsersModalActions
+const { getUserList } = searchUsersModalSelectors
+const { getUsers } = cacheUsersSelectors
+
+const DEBOUNCE_MS = 100
+
+const messages = {
+  title: 'New Message',
+  search: 'Search Users',
+  followsYou: 'Follows You',
+  followers: 'Followers'
+}
+
+const useStyles = makeStyles(({ spacing, palette, typography }) => ({
+  rootContainer: {
+    backgroundColor: palette.white,
+    flexGrow: 1
+  },
+  loadingSpinner: {
+    height: spacing(20),
+    width: spacing(20),
+    alignSelf: 'center'
+  },
+  searchContainer: {
+    marginTop: spacing(8),
+    marginHorizontal: spacing(2),
+    marginBottom: spacing(2)
+  },
+  searchIcon: {
+    width: spacing(5),
+    height: spacing(5)
+  },
+  searchInputContainer: {
+    paddingRight: spacing(4.5),
+    paddingVertical: spacing(4.5)
+  },
+  searchInputText: {
+    fontWeight: '700',
+    fontSize: typography.fontSize.large
+  },
+  profilePicture: {
+    height: spacing(18),
+    width: spacing(18)
+  },
+  userContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    padding: spacing(4),
+    borderBottomColor: palette.neutralLight4,
+    borderBottomWidth: 1
+  },
+  userNameContainer: {
+    flexGrow: 1,
+    display: 'flex',
+    flexDirection: 'column',
+    marginLeft: spacing(2.5)
+  },
+  userName: {
+    fontSize: typography.fontSize.small,
+    fontWeight: 'bold',
+    color: palette.neutral
+  },
+  followContainer: {
+    marginTop: spacing(1),
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'space-between'
+  },
+  handle: {
+    fontSize: typography.fontSize.small,
+    color: palette.neutral
+  },
+  followersContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center'
+  },
+  followersCount: {
+    fontWeight: '800',
+    marginHorizontal: spacing(1),
+    color: palette.neutralLight4,
+    fontSize: typography.fontSize.small
+  },
+  followers: {
+    color: palette.neutralLight4,
+    fontSize: typography.fontSize.small
+  },
+  iconUser: {
+    height: spacing(4),
+    width: spacing(4)
+  },
+  followsYouTag: {
+    fontSize: typography.fontSize.xxs,
+    fontWeight: '900',
+    letterSpacing: 0.64,
+    textTransform: 'uppercase',
+    color: palette.neutralLight4,
+    borderWidth: 1,
+    borderRadius: spacing(1),
+    borderColor: palette.neutralLight4,
+    paddingVertical: spacing(1),
+    paddingHorizontal: spacing(2)
+  }
+}))
+
+type SearchUsersScreenProps = {
+  debounceMs?: number
+  defaultUserList?: {
+    userIds: ID[]
+    loadMore: () => void
+    loading: boolean
+  }
+}
+
+export const ChatUserListScreen = (props: SearchUsersScreenProps) => {
```

Comment:
`SearchUsersScreenProps` but `ChatUserListScreen`?

I think it's probably good to separate this component out to make it useable by other features in the future, and have one that's specific to Chats that renders each user with the chats options/actions. That's how I did it for web

Reply:
Agree. Gonna merge this for now and make it generic in a follow-up.